docs/specs/servers/blockchain/blockchain-bundler-api.md

```diff
@@ -0,0 +1,51 @@
+# Blockchain API Bundlers interaction
+
+This API provides a bundlers JSON-RPC interaction. 
+The supported `userOperation` interaction schema is according to v0.7 ERC-4337 specification.
+
+## Make a bundler JSON-RPC request
+
+Used to make a JSON-RPC call to the designated bundler.
+
+`POST /v1/bundler?projectId={projectId}&chainId={chainId}&bundler={bundlerName}`
+
+* `projectId` - Required. The project identifier.
+* `chainId` - CAIP-2 chain identifier.
+* `bundlerName` - (Optional) Bundler name to use.
+
+The list of currently supported bundler names: 
+* `pimlico` - [Pimlico bundler](https://docs.pimlico.io/permissionless/reference/bundler-actions/sendUserOperation).
+
+#### Success response body:
+
+```typescript
+{
+    id:  number, // Unique identifier. Default is 1.
+    jsonrpc: string, // JSON-RPC version. Default 2.0
+    method: string, // ERC-4337 bundler operation methods. Please check supported methods below.
+    params: string[], // Array of string variables input for the method.
+}
+```
+
+List of supported `method` values:
+* `sendUserOperation` - Sends a user operation to the given EVM network.
```

Comment:
Since method names differ in different bundlers (not all, but some) we can unify them as I did here (and convert to the bundler-specific at the server-side) or pass the method through without any changes to the designated bundler.

Reply:
we need support for wallet_getCallsStatus and wallet_showCallsStatus, Im not aware if we need the others.

Reply:
https://eips.ethereum.org/EIPS/eip-5792

Reply:
@geekbrother please add `wallet_getCallsStatus` to the supported list

Reply:
I believe we also need to add: `wallet_sendCalls`, or how the client sends the batch to the designated bundler? I believe this should be also done by our endpoint? 
cc @llbartekll @jakubuid

Reply:
@ganchoradkov what's the implementation here? I thought that wallet is responsible for executing an atomic batch and this endpoint is for dapp to call for a status

Reply:
Updated the API: we will allow just `wallet_getCallsStatus` and `wallet_showCallsStatus` operations for now.